### PR TITLE
FIT-628: Removes preint deploy

### PR DESCRIPTION
**Pull request by edgibbs**
### Jira Story

- [FIT-628](https://osi-cwds.atlassian.net/browse/FIT-628)

## Description
Removing preint deploys

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [ ] I have ran lint/rubocop check for the changed/new files.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by mariamcws** [comment]
Looks good

File: Jenkinsfile [hunk 1]
```diff
@@ -340,13 +339,6 @@ def buildDocker() {
 
 def smokeTest(environment) {
   stage("Smoke Test on $environment") {
-    if (environment == 'preint') {
-      withEnv(["APP_URL=https://web.${environment}.cwds.io",
-             "FEATURE_SET=${FEATURE_SET}",
-             "CAPYBARA_DRIVER=${CAPYBARA_DRIVER}"]) {
```
Comment (TylerOSI) [inline Jenkinsfile]: do we need `FEATURE_SET` and `CAPYBARA_DRIVER` variables anymore? Are they used elsewhere?

Comment (edgibbs) [reply Jenkinsfile]: I'll take a look.

Comment (edgibbs) [reply Jenkinsfile]: Removed.

**Pull request merged**